apps/chat/src/components/Marketplace/TabRenderer.tsx

```diff
@@ -271,6 +288,25 @@ export const TabRenderer = ({ screenState }: TabRendererProps) => {
     ? modelsMap[detailsModelReference]
     : undefined;
 
+  const NoApplicationFound = () => (
+    <div className="flex grow flex-col items-center justify-center">
+      <IconMessage2 size={100} className="stroke-[0.2]" />
+      <span className="mt-5 text-lg font-semibold">{t('No agents')}</span>
+      <span className="mt-4 text-sm font-normal">
+        {t("You don't have any agents.")}
+      </span>
+    </div>
+  );
+
+  const NoResultsFoundMessage = () => (
+    <div className="flex grow flex-col items-center justify-center">
+      <NoResultsFound iconSize={100} className="gap-5 text-lg" />
+      <span className="mt-4 text-sm font-normal">
+        {t("Sorry, we couldn't find any results for your search.")}
+      </span>
+    </div>
+  );
+
```

Comment:
move this components out of the component

Reply:
What to do with "t" ?

Reply:
For each make its own 
const { t } = useTranslation(Translation.Marketplace); 
?

Reply:
At least these components must be out of this component